src/mapml-viewer.js

```diff
@@ -986,7 +989,256 @@ export class HTMLMapmlViewerElement extends HTMLElement {
       }
     });
   }
+  matchMedia(query) {
+    // useful features for maps: prefers-color-scheme, prefers-lang, projection, zoom, extent
+    const parsedQuery = parseMediaQueryList(query);
+
+    // less obviously useful: aspect-ratio, orientation, (device) resolution, overflow-block, overflow-inline
+
+    const map = this;
+    const features = {
+      'prefers-lang': {
+        type: 'discrete',
+        get values() {
+          return [navigator.language.substring(0, 2)];
+        }
+      },
+      'map-projection': {
+        type: 'discrete',
+        get values() {
+          return [map.projection.toLowerCase()];
+        }
+      },
+      'map-zoom': {
+        type: 'range',
+        valueType: 'integer',
+        canBeNegative: false,
+        canBeZero: true,
+        get extraValues() {
+          return {
+            min: 0,
+            max: map.zoom
+          };
+        }
+      },
+      'map-top-left-easting': {
```

Comment:
map-top-left-easting (i.e. > or < Double)
map-bottom-right-easting (i.e. > or < Double)
map-bottom-right-northing (i.e. > or < Double)
map-top-left-northing (i.e. > or < Double)

May be a little hard for users to understand/utilize the extent feature queries. A nice graphic in the docs or something similar would be needed to explain the extents and how to properly use them via media queries